README.md

```diff
@@ -207,23 +220,67 @@ More information can be found [here](http://www.diveintopython3.net/strings.html
 | pyVersion        | Returns the version of Python                      | `pyVersion()`                                                        |
 
 
-# Usage Examples   
+# Wrapping python packages
+
+While using the original PythonInR to wrap a python package in R, automatically generates R functions and documentation, we added a few helper functions in out version of PythonEmbedInR:
+
+## Determine args and kwargs (determineArgsAndKwArgs)
```

Comment:
determineArgsAndKwArgs() is really just a small utility which is part of the larger framework for generating wrapper functions.  Do you think it makes more sense to add the larger framework here?  determineArgsAndKwArgs() would then just be an internal method and there'd be no need for extensive documentation.

Reply:
Note that we didn't take synapseclient and wrap every functions & classes in that python package. We cherry pick functions and classes under synapseclient.Synapse (+ synapseclient.table.Table) for synapser package. We then cherry pick synapseutils for synapserutils.